### PR TITLE
Techassi Translation

### PR DESCRIPTION
**Pull request by Techassi**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _articles/de-DE/best-practices.md [hunk 1]
```diff
@@ -249,31 +249,31 @@ Aber, wenn Standards zu kompliziert sind, erhöhen sie die Schwierigkeit, beizut
 
 Wenn du unsicher bist, welche Erweiterungen du benutzen sollst, schau dir einfach andere populäre Projekte an, besonders die, die im selben Bereich aktiv sind. Wie sieht zum Beispiel bei anderen Node Modulen der Beitrags-Prozess aus? Ähnliche Wege und Werkzeuge machen den Prozess einfacher, auch für deine Zielgruppe.
 
-## It's okay to hit pause
+## Es ist ok eine Pause einzulegen
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: `OK`

File: _articles/de-DE/best-practices.md [hunk 2]
```diff
@@ -249,31 +249,31 @@ Aber, wenn Standards zu kompliziert sind, erhöhen sie die Schwierigkeit, beizut
...
 
-Open source work once brought you joy. Maybe now it's starting to make you feel avoidant or guilty.
+Open Source Arbeit brachte dir Freude. Vielleicht fühlst du dich jetzt dadurch ausweichend oder schuldig.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: `Open-Source`

File: _articles/de-DE/best-practices.md [hunk 3]
```diff
@@ -249,31 +249,31 @@ Aber, wenn Standards zu kompliziert sind, erhöhen sie die Schwierigkeit, beizut
...
 
-Perhaps you're feeling overwhelmed or a growing sense of dread when you think about your projects. And meanwhile, the issues and pull requests pile up.
+Möglicherweise fühlst du dich überwältigt oder deine Furcht wächst immer weiter, wenn du an deine Projekte denkst. Und währenddessen stapeln sich die Probleme und Pull Requests.
 
-Burnout is a real and pervasive issue in open source work, especially among maintainers. As a maintainer, your happiness is a non-negotiable requirement for the survival of any open source project.
+Burnout ist ein reales und um sich greifendes Problem bei der Open Source Arbeit, besonders unter den Instandhaltern. Als Entwickler ist deine Freude am Projekt ausschlaggebend für den Fortbestand jedes Open Source Projekts.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: `Instandhaltern` => `Projektleitern`

File: _articles/de-DE/best-practices.md [hunk 4]
```diff
@@ -249,31 +249,31 @@ Aber, wenn Standards zu kompliziert sind, erhöhen sie die Schwierigkeit, beizut
...
 
-Although it should go without saying, take a break! You shouldn't have to wait until you feel burned out to take a vacation. @brettcannon, a Python core developer, decided to take [a month-long vacation](http://www.snarky.ca/why-i-took-october-off-from-oss-volunteering) after 14 years of volunteer OSS work.
+Obwohl es selbstverständlich sein sollte: Nimm dir eine Pause! Du solltest nicht warten bis du dich erschöpfst fühlst und dir einen Urlaub nehmen musst. @brettcannon, ein Python Kern Entwickler, entschied sich einen [gesamten Monat Urlaub](http://www.snarky.ca/why-i-took-october-off-from-oss-volunteering) zu nehmen, und das nach 14 Jahren freiwilliger Arbeit in der Open Source Gemeinschaft.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: append ` (Englisch)` to the link title, please.

File: _articles/de-DE/best-practices.md [hunk 5]
```diff
@@ -249,31 +249,31 @@ Aber, wenn Standards zu kompliziert sind, erhöhen sie die Schwierigkeit, beizut
...
+So wie bei jeder Art von Arbeit hilft es regelmäßige Pausen einzulegen, um einen klaren Kopf zu behalten und glücklich und begeistert an deine Arbeit heran zu gehen.
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars1.githubusercontent.com/u/36432?v=3&s=400" class="pquote-avatar" alt="avatar">
-  In maintaining WP-CLI, I've discovered I need to make myself happy first, and set clear boundaries on my involvement. The best balance I've found is 2-5 hours per week, as a part of my normal work schedule. This keeps my involvement a passion, and from feeling too much like work. Because I prioritize the issues I'm working on, I can make regular progress on what I think is most important.
+  Bei dem Instandhalten von WP-CLI, merkte ich, dass ich mich selbst glücklich machen muss und klare Grenzen meines Mitwirkens zu ziehen. Die beste Balance ist es, zwischen 2 und 5 Stunden in der Wochen, neben der regulären Arbeit, zu arbeiten. So bleibt mein Engagement eine Passion und fühlt sich nicht wie Arbeit an. Ich konzentriere mich auf die Probleme, an denen ich gerade arbeite. So kann ich regelmäßigen Fortschritt machen und zwar dort wo es am wichtigsten ist.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: `Instandhaltern` => `Projektleitern`

**Comment by Techassi** [comment]
Fixed and improved some passages.

File: _articles/de-DE/best-practices.md [hunk 6]
```diff
@@ -249,21 +249,21 @@ Aber, wenn Standards zu kompliziert sind, erhöhen sie die Schwierigkeit, beizut
...
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars1.githubusercontent.com/u/36432?v=3&s=400" class="pquote-avatar" alt="avatar">
-  Bei dem Instandhalten von WP-CLI, merkte ich, dass ich mich selbst glücklich machen muss und klare Grenzen meines Mitwirkens zu ziehen. Die beste Balance ist es, zwischen 2 und 5 Stunden in der Wochen, neben der regulären Arbeit, zu arbeiten. So bleibt mein Engagement eine Passion und fühlt sich nicht wie Arbeit an. Ich konzentriere mich auf die Probleme, an denen ich gerade arbeite. So kann ich regelmäßigen Fortschritt machen und zwar dort wo es am wichtigsten ist.
+  Bei dem Projektleiter von WP-CLI, merkte ich, dass ich mich selbst glücklich machen muss und klare Grenzen meines Mitwirkens zu ziehen. Die beste Balance ist es, zwischen 2 und 5 Stunden in der Woche, neben der regulären Arbeit, zu arbeiten. So bleibt mein Engagement eine Passion und fühlt sich nicht wie Arbeit an. Ich konzentriere mich auf die Probleme, an denen ich gerade arbeite. So kann ich regelmäßigen Fortschritt machen und zwar dort wo es am wichtigsten ist.
```
Comment (jens1o) [inline _articles/de-DE/best-practices.md]: `Bei dem Leiten von WP-CLI`? I'm struggeling...

**Comment by Techassi** [comment]
Small update.

**Comment by jens1o** [comment]
perfect!

**Comment by jens1o** [comment]
I'm going to translate the few lines beneath it, and then the page is done! :)

**Comment by Techassi** [comment]
Good job. I am going to continue with the next page.

**Comment by jens1o** [comment]
Which one? I actually planned to continue with the https://opensource.guide/finding-users/ page.

Feel free to do a PR after each day you've worked on.

**Comment by Techassi** [comment]
I am going to translate https://github.com/Techassi/de-opensource.guide/blob/gh-pages/_articles/de-DE/getting-paid.md page.

**Pull request merged**